### ISSUE TITLE
Moral Expression Detection and Classification

### ISSUE THREAD

**Issue by iulusoy** [ssciwr/moralization#51]
-Train/Test Data:
-  Version 1: KAT2-Moralwerte und KAT2-Subjektive_Ausdrücke als eine Kategorie zusammenlegen
-  Version 2: getrennt für KAT2-Moralwerte und KAT2-Subjektive_Ausdrücke

Task: Gegeben Class 1 von Task 1:
-  A: Extraktion aller Wörter bzw. Mehrworteinheiten, die moralische Werte und/oder subjektive Ausdrücke beinhalten
-  B: Klassifikation der extrahierten Wörter bzw. Mehrworteinheiten:
    -  13 Klassen: Care, Harm, Fairness, Cheating, Loyalty, Betrayal, Autority, Subversion, Purity, Degradation, Liberty, Oppression, OTHER
-  Multi-Label Setup (e.g. Sigmoid Layer (instead of softmax) as last output layer)

Relevante NLP-Tools/Libraries:
-  Chunking/Dependency Parsing als preprocessing?
-  Systems pretrained on MFT Detection, finetuned on our data
-  string match Morallexika (ggf. Lemmatization als pretraining)    
-   …?
